server/service.go

```diff
@@ -200,6 +206,22 @@ func (m *BoostService) sendValidatorRegistrationsToRelayMonitors(payload []types
 	}
 }
 
+func (m *BoostService) sendAuctionTranscriptToRelayMonitors(transcript AuctionTranscript) {
+	log := m.log.WithField("method", "sendAuctionTranscriptToRelayMonitors")
+	for _, relayMonitor := range m.relayMonitors {
+		go func(relayMonitor *url.URL) {
+			url := GetURI(relayMonitor, pathAuctionTranscript)
+			log := log.WithField("url", url)
+			_, err := SendHTTPRequest(context.Background(), m.httpClientGetPayload, http.MethodPost, url, UserAgent(""), transcript, nil)
```

Comment:
do we want to use this particular `http.Client` here? I see this is logically under the `getPayload` path but im not sure that is what we want long term... could be fine for now and just something to keep in mind if it becomes a hotspot